README.md

```diff
@@ -100,9 +100,14 @@ Click on any of the items below to expand the corresponding answer.
 <details>
   <summary>I don't know what my Roomba's IP address is</summary>  
 
-  >In order to find your Roomba's IP address, you'll need to open up your Router Admin Panel and look for a list of devices. Once you identify the Roomba, you should see an associated IP address, however, this process will be different for each type of router.
```

Comment:
I'm not sure we should remove the previous instructions. I think that can be an Alternatively… paragraph?